src/main/java/com/company/SpringApplication/Publishers/PublisherDashboard.java

```diff
@@ -0,0 +1,17 @@
+package com.company.SpringApplication.Publishers;
```

Comment:
package names should be lower case, you can check java code conventions for code formatting